web/client-api/src/main/java/io/deephaven/web/client/api/JsWorkerHeapInfo.java

```diff
@@ -0,0 +1,31 @@
+package io.deephaven.web.client.api;
+
+import io.deephaven.javascript.proto.dhinternal.io.deephaven.proto.console_pb.HeapInfo;
+import jsinterop.annotations.JsProperty;
+
+public class JsWorkerHeapInfo {
+    private long maximumHeapSize;
+    private long freeMemory;
+    private long totalHeapSize;
+
+    public JsWorkerHeapInfo(HeapInfo heapInfo) {
+        this.maximumHeapSize = Long.parseLong(heapInfo.getMaximumHeapSize());
+        this.freeMemory = Long.parseLong(heapInfo.getFreeMemory());
+        this.totalHeapSize = Long.parseLong(heapInfo.getTotalHeapSize());
+    }
+
+    @JsProperty
+    public long getMaximumHeapSize() {
```

Comment:
don't directly expose long to JS, you'll at best get compiler warnings and at worst sometimes see objects instead of numbers.

instead return double if you're okay with truncating the value, or clean it up with LongWrapper or String

Reply:
Yea truncating the value is probably fine. I'll need to fix this on Enterprise as well.

In testing it was just returning a number,